resources/views/store/checkout.blade.php

```diff
@@ -37,21 +37,10 @@
             @endif
 
             @if ($order->isProcessing())
-                @php
-                    $cancelLink = Html::link(
-                        route('store.checkout.destroy'),
-                        trans('store.checkout.pending_checkout.link_text'),
-                        [
-                            'data-method' => 'DELETE',
-                            'data-confirm' => trans('common.confirmation'),
-                            'data-remote' => '1',
-                        ]
-                    );
-                @endphp
                 <div class="alert alert-danger">
                     <p>
                         {{ trans('store.checkout.pending_checkout.line_1') }}<br>
-                        {!! trans('store.checkout.pending_checkout.line_2', ['link' => $cancelLink]) !!}
+                        {{ trans('store.checkout.pending_checkout.line_2') }}
```

Comment:
no way to cancel anymore? And then there's nothing routing to `Checkout#destroy` (which probably needs update so it can cancel any processing orders).

Reply:
The current method of the user cancelling an inprogress checkout is no longer supported since being unable to handle multiple checkouts separately was one of the reasons it was added. Some kind of stale cart cleaner is being planned to replace it, but the details of that are tbd.

A side-effect is`failCheckout()` is only triggered by Paypal checkouts at the moment until I figure out where to put it for Xsolla checkouts.

Reply:
> Some kind of stale cart cleaner is being planned to replace it, but the details of that are tbd.

More like stale order cleaner because it's not cart anymore.